javascript/binding.gyp

```diff
@@ -1,17 +1,25 @@
 {
     "targets": [{
-    	"includes": [
-    	    "auto.gypi",
+        "includes": [
+            "auto.gypi",
             "final-flags.gypi"
-    	],
+        ],
 
-    	"sources": [
-            "<!@(ls -1 sources/yoga/*.cpp)",
```

Comment:
could you use `shx` here, too? The reason I am asking is that we have a growing number of lists of files that need to be kept up to date.

Reply:
@simast are you still keen to get this resolved for us?